<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1568</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
/*background-color: red;*/
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: black;
font: bold 14px;
min-width: 10em;
padding: 6px;
}
QPushButton:pressed {
background-color: rgb(0, 0, 0);
border-style: inset;
}

QGroupBox {
border: 1px solid gray;
border-color: #FF17365D;
margin-top: 27px;
font-size: 14px;
border-bottom-left-radius: 15px;
border-bottom-right-radius: 15px;
}

QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top center;
border-top-left-radius: 15px;
border-top-right-radius: 15px;
padding: 5px 150px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}
QLabel
{
border: 2px solid black;
border-radius: 4px;
padding: 2px;
background-image: rgb(255,255,255)
}
QLineEdit {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
/*background: yellow;*/
selection-background-color: darkgray;
}
/*QTableView QTableCornerButton::section {
background: ;
border: 2px outset red;
}*/
QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #C2C7CB;
position: absolute;
top: -0.5em;
}

QTabWidget::tab-bar {
alignment: center;
}
background-color: rgb(210, 210, 210);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>1251</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>1061</width>
        <height>611</height>
       </rect>
      </property>
      <property name="title">
       <string>Donnner vos informations </string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom         : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prénom     :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nomres">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>110</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenomres">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>160</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_numres">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>211</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>210</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero de téléphone :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID de résérvation       :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idres">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>260</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prixres">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>301</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>310</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix                          : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>360</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de Résérvation   :</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>729</x>
         <y>150</y>
         <width>311</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>QR code </string>
       </property>
       <widget class="QLabel" name="qr_code">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>191</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>410</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Date                         :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouterres">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>550</y>
         <width>186</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ignorerres">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>550</y>
         <width>186</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Ignorer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cb_typeres">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>360</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_dateres">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>420</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Calendrier">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>410</y>
         <width>186</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Calendrier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>mettre a jours </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>671</width>
        <height>611</height>
       </rect>
      </property>
      <property name="title">
       <string>Les Résérvations </string>
      </property>
      <widget class="QTableView" name="tab_res">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>651</width>
         <height>501</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>560</y>
         <width>186</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Version PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>30</y>
        <width>551</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier une résérvation </string>
      </property>
      <widget class="QLineEdit" name="le_idmres">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>170</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID de résérvation : </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>330</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom                    :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom               :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenommres">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>90</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nommres">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nummres">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>130</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Num                   :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prixmres">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>210</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix                    :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_typemres">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>250</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Type                  :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Date                   :</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_datemres">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>290</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>410</y>
        <width>551</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>trier/rechercher </string>
      </property>
      <widget class="QPushButton" name="trires">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>chercher : </string>
       </property>
      </widget>
      <widget class="QPushButton" name="triertype">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>80</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier type</string>
       </property>
      </widget>
      <widget class="QPushButton" name="triernum">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>160</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier Num</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cherherres">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="chercherres">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>186</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chercherrestype">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>186</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chercherresnom">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>186</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cherherrest">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cherherresn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Effacer une résérvation </string>
     </attribute>
     <widget class="QPushButton" name="supprmierres">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>340</y>
        <width>186</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner ID de résérvation : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_suppres">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>160</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>mailing</string>
     </attribute>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>620</y>
        <width>851</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>recipient</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>851</width>
        <height>381</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1568</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
